integration/e2emimir/services.go

```diff
@@ -212,6 +212,48 @@ func NewSingleBinary(name string, flags map[string]string, options ...Option) *M
 	)
 }
 
+func NewReadInstance(name string, flags map[string]string, options ...Option) *MimirService {
```

Comment:
Not sure about my terminology here - is "read instance" (or "write instance" or "backend instance") the term we use? 

Or is another term preferred? Something like "reader" or "read component" or "read service" perhaps?

Reply:
Your terminology is correct:
- Component: a Mimir deployable component, e.g. read
- Instance: a replica of a given component (e.g. a replica of read), which is what we're creating here
- Service: we try to not use it because of misunderstanding with Kubernetes services